scrapers/stackoverflow.js

```diff
@@ -0,0 +1,61 @@
+var unirest = require("unirest");
+var question = "";
+for (i = 2; i < process.argv.length; i++) {
+    question += process.argv[i] + " ";
+}
+var url = "https://api.stackexchange.com/2.2/search/advanced?key=jAAMPr3uTSgYulMJ3LY5Hw((&order=desc&sort=activity&q=" + question + "&answers=1&site=stackoverflow";
+unirest.get(url)
+    .end((response) => {
+        var response = JSON.parse(response.raw_body);
+        var QandA = new Array();
+        if (response.quota_remaining > 0) {
+            j = 0;
+            for (i = 0; i < response.items.length; i++) {
+
```

Comment:
Sorry for late reply. As there were too many comments, I probably missed this one..
So if the length is 4 ...then final array will be printed only when it has received 4 items(i.e. JSONObjects).

getQuestiomAndAnswers() method will be called 4 times. For counting there is a counter named "j" . You can check it. Still if there is any issue then let me know fast.
As after getting all the reviews only I will start making changes.